code/modules/unit_tests/turf_icons.dm

```diff
@@ -0,0 +1,45 @@
+/// Makes sure turf icons actually exist. :)
+/datum/unit_test/turf_icons
+
+/datum/unit_test/turf_icons/Run()
+	for(var/turf/turf_path as anything in (subtypesof(/turf) - typesof(/turf/closed/mineral)))
+
+		var/icon_state = initial(turf_path.icon_state)
+		var/icon_file = initial(turf_path.icon)
+		if(isnull(icon_state) || isnull(icon_file))
+			continue
+		if(!(icon_state in icon_states(icon_file)))
+			TEST_FAIL("[turf_path] using invalid icon_state - \"[icon_state]\" in icon file, '[icon_file]")
+
+	for(var/turf/closed/mineral/turf_path as anything in typesof(/turf/closed/mineral)) //minerals use a special (read: snowflake) MAP_SWITCH definition that changes their icon based on if we're just compiling or if we're actually PLAYING the game.
+
+		var/icon_state = initial(turf_path.icon_state)
+		var/icon_file = 'icons/turf/mining.dmi'
+		if(isnull(icon_state))
+			continue
+		if(!(icon_state in icon_states(icon_file)))
+			TEST_FAIL("[turf_path] using invalid icon_state - \"[icon_state]\" in icon file, '[icon_file]")
+
+	var/turf/initial_turf_type = run_loc_floor_bottom_left.type
+
+	for(var/turf/open/open_turf_path as anything in subtypesof(/turf/open))
```

Comment:
Ah! Found it & it makes sense now. Seems you guys blacklisted that specific turf type in the create_and_destroy unit test due to it relying on SSLiquids being init'd, otherwise you had the same issue over there.
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/6209658/414cbdc3-df5d-4af0-abf0-6beacec903ff)
We can just do that here as well.

 
```suggestion
	var/list/ignored_types = list()
	//SKYRAT EDIT START
	ignored_types += typesof(/turf/open/openspace/ocean)
	ignored_types += typesof(/turf/open/floor/plating/ocean)
	// SKYRAT EDIT END

	for(var/turf/open/open_turf_path as anything in (subtypesof(/turf/open) - ignored_types))
```

Rest of the code not in the SKYRAT edit block was PR'd upstream.
https://github.com/tgstation/tgstation/pull/75596


I've reviewed all the icons caught by the unit test and can confirm they're all positive catches by the test (meaning they're missing icons.) A couple of them were made in this PR and need to be added to your dmi files.